src/vs/workbench/browser/parts/views/viewsService.ts

```diff
@@ -239,7 +240,9 @@ export class ViewsService extends Disposable implements IViewsService {
 
 	getFocusedViewName(): string {
 		const viewId: string = this.contextKeyService.getContextKeyValue(FocusedViewContext.key) ?? '';
-		return this.viewDescriptorService.getViewDescriptorById(viewId.toString())?.name?.value ?? '';
+		const context = this.contextKeyService.getContext(document.activeElement as HTMLElement);
```

Comment:
I think this is a hack that leverages context keys to figure out state. But I think we should use proper services here to check and not rely on a context key to be set.

Reply:
You are right, but that will require adding some things to the services.

Reply:
figured it out